app/views/events/index.html.erb

```diff
@@ -0,0 +1,44 @@
+
+<div id="calendar" class="row"></div>
+
+<script>
+  $(document).ready(function () {
+   var events_arr = [];
+   <% @events.first.each do |event| %>
+    events_arr.push({
+      title: "<%= event.title %>",
+      description: "<%= event.description %>",
+      datetime: new Date(<%= event.event_date.year%>, <%= event.event_date.month %> -1, <%= event.event_date.day%>, <%= event.event_date.strftime("%H") %>)
+    })
+  <% end %>
+  <% @events.last.each do |key,val| %>
+    events_arr.push({
+      title: "Public Holiday",
+      description: "This is a public holiday",
+      datetime: new Date(<%= Date.parse(@events.last[key]["holiday_date_#{key}"]).year %>, <%= Date.parse(@events.last[key]["holiday_date_#{key}"]).month %> -1, <%= Date.parse(@events.last[key]["holiday_date_#{key}"]).day%>, <%= Date.parse(@events.last[key]["holiday_date_#{key}"]).strftime("%H") %>)
+    })
+    <% if @events.last[key]["every_year_#{key}"] %>
```

Comment:
write a helper function for this

Reply:
that's jquery has to be done here I guess